packages/usa-accordion/src/usa-accordion.stories.js

```diff
@@ -21,9 +28,47 @@ Bordered.args = BorderedContent;
 export const Multiselectable = Template.bind({});
 Multiselectable.args = MultiContent;
 
-export const TestIcons = TestTemplate.bind({});
-TestIcons.args = {
+export const Test = TestTemplate.bind({});
+Test.args = {
   ...DefaultContent,
-  ...HeaderContent,
   ...BannerContent,
+  header: {
+    ...HeaderContent,
+    site_title: "Basic header",
+    nav: {
+      ...HeaderContent.nav,
+      aria_label: "Test navigation",
+    },
+    search: null,
+  },
+  headerExtended: {
+    ...HeaderExtendedContent,
+    site_title: "Extended header",
+    nav: {
+      ...HeaderExtendedContent.nav,
+      aria_label: "Test extended navigation",
+    },
+    search: null,
+    secondaryNav: false,
+  },
+  headerMegamenu: {
+    ...HeaderMegamenuContent,
+    site_title: "Megamenu header",
+    nav: {
+      ...HeaderMegamenuContent.nav,
+      aria_label: "Test megamenu navigation",
+    },
+    search: null,
+    secondaryNav: false,
+  },
+  headerMegamenuExtended: {
+    ...HeaderExtendedMegamenuContent,
+    site_title: "Extended megamenu header",
+    nav: {
+      ...HeaderExtendedMegamenuContent.nav,
+      aria_label: "Test extended megamenu navigation",
+    },
+    search: null,
+    secondaryNav: false,
+  },
```

Comment:
A lot of overrides here to have a simple menu. Header needs refactoring to get it in a more customizable state.